tests/scripts/prepare-bellecour-fork-for-tests.js

```diff
@@ -18,7 +18,7 @@ const PROD_WORKERPOOL_OWNER_WALLET =
   '0x1Ff6AfF580e8Ca738F76485E0914C2aCaDa7B462';
 const DEBUG_WORKERPOOL = '0xdb214a4a444d176e22030be1ed89da1b029320f2'; // 'debug-v8-bellecour.main.pools.iexec.eth';
 const PROD_WORKERPOOL = '0x0e7bc972c99187c191a17f3cae4a2711a4188c3f'; // 'prod-v8-bellecour.main.pools.iexec.eth';
-const ORACLE_DAPP_ADDRESS = '0xd11f5d70f8817add3d9e15d316911e6a4d699f79'; // oracle-factory.apps.iexec.eth
```

Comment:
using hardcoded addresses is not ideal as ENS will change and test will break.
here is how we fixed it in another project
https://github.com/iExecBlockchainComputing/web3mail-sdk/blob/00dcc4976d6152020e33d3d4128a35e82c25e322/tests/scripts/prepare-bellecour-fork-for-tests.js#L22-L42

Reply:
commit : e6967f0e19fee872ca41e5c11397b5338c6d9b79